repo-scripts/size-analysis/bundle/rollup.ts

```diff
@@ -36,9 +36,7 @@ export async function bundleWithRollup(
   };
 
   if (moduleDirectory) {
-    resolveOptions.customResolveOptions = {
-      moduleDirectory
-    };
+    resolveOptions.moduleDirectories = [moduleDirectory];
```

Comment:
Nice! How did you catch the problem?

Reply:
I mean, it errors if you try to run it with the upgraded plugin, so I checked the changelog for breaking changes and this was there.